### PR TITLE
Task_dev_3

### PR DESCRIPTION
**Pull request by Marymaryyyyyy**
Add files

### CODE REVIEW

File: Converter.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    /// This class converts the number in other notation system.
+    /// </summary>
+    class Converter
+    {
+        private string convertedValue;            
+        
+        /// <summary>
+        /// This class converts the number in other notation system.
+        /// </summary>
+        /// <param name="number">input number</param>
```
Comment (IgorKheidorov) [inline Converter.cs]: Very bad unclear name, what is it

File: EntryPoint.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            int number;
+            int convertedToValue;
+
+            if (args.Length > 2)
+            {
```
Comment (IgorKheidorov) [inline EntryPoint.cs]: Rework to Parse and exception handling as it was agreed in chat

File: EntryPoint.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                bool checkNumber = Int32.TryParse(args[0], out number);
+                bool checkFormatBase = Int32.TryParse(args[1], out convertedToValue);
```
Comment (IgorKheidorov) [inline EntryPoint.cs]: Bag

File: EntryPoint.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                {
+                    Converter converter = new Converter();
+                    Console.WriteLine("Output: {0}", converter.ConvertTo(number, convertedToValue));
+                }
+                else
+                {
+                    Console.WriteLine("Error! Incorrect format of the input data!");
+                }
+            }
+            else
```
Comment (IgorKheidorov) [inline EntryPoint.cs]: Code style

File: Converter.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Task_dev_3
+{
+    /// <summary>
+    /// This class converts the number in other notation system.
+    /// </summary>
```
Comment (IgorKheidorov) [inline Converter.cs]: convertor

File: Converter.cs [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+                int rest = number % convertedToValue;
+                number = number / convertedToValue;
+                convertedValue = ChooseLetter(rest) + convertedValue;
+            }
+            while (number > 0);
+            return convertedValue;
+        }
+
+        private char ChooseLetter(int rest)
+        {
```
Comment (IgorKheidorov) [inline Converter.cs]: use const

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: Converter.cs [hunk 7]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Task_dev_3
+{
+    /// <summary>
+    ///Converter converts the number in other notation system.
+    /// </summary>
+    class Converter
+    {
+        private string convertedValue;
```
Comment (AlexanderSoroka) [inline Converter.cs:10]: useless field

File: Converter.cs [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        /// <param name="convertedToValue">new number notation</param>
+        /// <returns>output number of the new notation system </returns>
+        public string ConvertTo(int inputNumber, int convertedToValue)
+        {
+            do
+            {
+                int rest = inputNumber % convertedToValue;
+                inputNumber = inputNumber / convertedToValue;
+                convertedValue = ChooseSymbol(rest) + convertedValue;
+            }
```
Comment (AlexanderSoroka) [inline Converter.cs:25]: codestyle violation

**Review by AlexanderSoroka — APPROVED** [review]
Not accepted as it contains changes which don't relate to the task

**Pull request merged**